test/acsengine-features.groovy

```diff
@@ -37,6 +37,9 @@ node {
                 env.SUBSCRIPTION_ID="${SUBSCRIPTION_ID}"
                 env.CLUSTER_SERVICE_PRINCIPAL_CLIENT_ID="${CLUSTER_SERVICE_PRINCIPAL_CLIENT_ID}"
                 env.CLUSTER_SERVICE_PRINCIPAL_CLIENT_SECRET="${CLUSTER_SERVICE_PRINCIPAL_CLIENT_SECRET}"
+                if(CUSTOM_HYPERKUBE_SPEC) {
```

Comment:
also, PR bot will not be able to catch it cuz it doesn't use the groovy pipeline.